vm/src/stdlib/posix.rs

```diff
@@ -1278,7 +1278,7 @@ pub mod module {
         fn spawn(self, spawnp: bool, vm: &VirtualMachine) -> PyResult<libc::pid_t> {
             use crate::TryFromBorrowedObject;
 
-            let path = CString::new(self.path.into_bytes())
+            let path = CString::new(self.path.clone().into_bytes())
```

Comment:
```suggestion
            let path = CString::new(self.path.as_bytes())
```
is this working?

Reply:
No, we don't have `as_bytes` for `PyPathLike` but IMO it makes sense to implement it.

Reply:
Probably not. `Path` doesn't look like to be borrowable as bytes.

Reply:
But I found another utility to do this job
```suggestion
            let path = self.path.into_cstring()?;
```

Reply:
We need to clone for this to work.